src/app/components/tag/TagInput.js

```diff
@@ -8,6 +8,19 @@ import CheckContext from '../../CheckContext';
 import { units, Row } from '../../styles/js/shared';
 import { createTag } from '../../relay/mutations/CreateTagMutation';
 import { stringHelper } from '../../customHelpers';
+import { makeStyles } from '@material-ui/core/styles';
+
+const TagInputStyle = makeStyles ({
+ 
+  paddingStyle: {
+    padding: units(2),
+  },
+  
+  buttonStyle: {
+    marginLeft: 'auto',
+  },                           
+
+});
```

Comment:
We're trying to use the `Box` component from @material-ui here. https://material-ui.com/components/box/